File: java/leap_year.java
import java.util.Scanner;

public class leap_year
{
	public static void main(String args[])
	{
		Scanner year = new Scanner(System.in); 		//create a object for Scanning value
		System.out.println("\n\nEnter the year: ");
		int yr = year.nextInt();			//yr varialble, year object and nextInt() function

		if(yr % 4 == 0)		// Condition to find leap year
		{
			System.out.println("Given year is leap.");
		}
		else
		{
			System.out.println("Given year is not leap.");
		}
	}
}

/*
OUTPUT

Enter the year:
2020
Given year is leap.


Enter the year:
2018
Given year is not leap.


Enter the year:
2024
Given year is leap.
*/

File: cpp/leap_year.cpp
#include<iostream>
using namespace std;

int main()
{
	int yr;

	cout<<"\n\nEnter the year:";
	cin>>yr;

	yr = yr % 4;		//fORMULA TO FIND LEAP YEAR

	if(yr == 0)
	{
	 cout<<"Given year are leap.";
	}
	else
	{
	 cout<<"Given year are not leap.\n\n";
	}
}
/*
OUTPUT

Enter the year:2020
Given year are leap.


Enter the year:2022
Given year are not leap.


Enter the year:2018
Given year are not leap.


Enter the year:2019
Given year are not leap.


Enter the year:2016
Given year are leap.
*/
